packages/dds/map/src/directory.ts

```diff
@@ -781,17 +781,21 @@ export class SharedDirectory
 						// guaranteed during the serialization process. As a result, it is only essential to utilize the
 						// "fake" client sequence number to signify the loading order, and there is no need to retain
 						// the actual client sequence number at this point.
-						if (createInfo !== undefined && createInfo.csn > -1) {
-							// If csn is -1, then initialize it with 0, otherwise we will never process ops for this
-							// sub directory. This could be done at serialization time too, but we need to maintain
-							// back compat too and also we will actually know the state when it was serialized.
+						if (createInfo !== undefined && createInfo.csn > 0) {
 							if (!tempSeqNums.has(createInfo.csn)) {
 								tempSeqNums.set(createInfo.csn, 0);
 							}
 							let fakeClientSeq = tempSeqNums.get(createInfo.csn) as number;
 							seqData = { seq: createInfo.csn, clientSeq: fakeClientSeq };
 							tempSeqNums.set(createInfo.csn, ++fakeClientSeq);
 						} else {
+							/**
```

Comment:
I assume getting here with createInfo  === undefined is same as csn === -1 (the comment could be a bit more inclusive if that's the case)

Reply:
Oh, never mind, it was existing code (but it would be still nice to somehow acknowledge undefined case)